spring-kafka-test/src/main/java/org/springframework/kafka/test/assertj/KafkaConditions.java

```diff
@@ -50,6 +51,15 @@ private KafkaConditions() {
 	}
 
 	/**
+	 * @param type the type of timestamp
+	 * @param value the timestamp.
+	 * @return a Condition that matches the timestamp value in a consumer record.
+	 */
+	public static Condition<ConsumerRecord<?, ?>> timestamp(TimestampType type, long value) {
```

Comment:
We might not use `KafkaMatchers` internally, but `spring-kafka-test` is a test utilities set exactly for end-users.
And there is no guaranty how they use AssertJ or stay with the Hamcrest.
Surely it is like you use `spring-integration-test` or well-known `spring-test` or new cool `spring-boot-test`.